static_analyzer/engine/edge_builder.py

```
@@ -306,15 +323,17 @@ def _resolve_definitions(
 
         for batch_start in range(0, len(call_sites), batch_size):
             batch = call_sites[batch_start : batch_start + batch_size]
-            queries = [(file_path, line, col) for line, col in batch]
+            queries = [(file_path, site.line - 1, site.column - 1) for site in batch]
 
             try:
                 results, _ = ctx.lsp.send_definition_batch(queries)
             except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)